release/index.md

```diff
@@ -17,7 +17,7 @@ The following table describes the status of Magento software availability and wh
 | **Product Recommendations**                       | Early access January 2020                                                                                                                                                                                                                                                                            | [Email request](mailto:mailto:magento-product-recs-feedback@adobe.com)                            |
 | **PWA Studio**                                    | Available now                                                                                                                                                                                                                                                                                        | [Documentation](http://pwastudio.io) and [GitHub](https://github.com/magento-research/pwa-studio) |
 | **Amazon Sales Channel 3.0.1**                    | Available now for {{site.data.var.ece}} 2.2.4+ and 2.3.x (US, Canada, Mexico, UK)                                                                                                                                                                                                                    | [Magento Marketplace](https://marketplace.magento.com/magento-module-amazon.html)                 |
-| **Google Shopping ads Channel 3.0.1**             | Available now for {{site.data.var.ece}} 2.2.4+ and 2.3.x <br>Bundled in Magento starting with 2.3.3, but still available on the Marketplace | [Magento Marketplace](http://marketplace.magento.com/magento-google-shopping-ads.html)            |
+| **Google Shopping ads Channel 3.0.2**             | Available now for {{site.data.var.ece}} 2.2.4+ and 2.3.x <br>Bundled in Magento starting with 2.3.3, but still available on the Marketplace | [Magento Marketplace](http://marketplace.magento.com/magento-google-shopping-ads.html)            |
```

Comment:
```suggestion
| **Google Shopping ads Channel 3.0.2**             | Google Shopping ads Channel 3.0.2 is available now for {{site.data.var.ece}} 2.2.4+ and 2.3.x. <br>It is bundled in Magento starting with 2.3.3, but still available on the Marketplace. | [Magento Marketplace](http://marketplace.magento.com/magento-google-shopping-ads.html)            |
```

Reply:
to keep with the formatting of above line items, declining this suggestion